packages/intrepid2/src/Shared/Intrepid2_Utils.hpp

```diff
@@ -383,6 +383,46 @@ namespace Intrepid2 {
       return ViewTypeWithLayout(label,dims...,derivative_dimension);
     }
   }
+
+  template< bool B, class T >
+  using enable_if_t = typename std::enable_if<B,T>::type;
```

Comment:
`Trilinos` requires `C++14` by now (because `Kokkos` does) so that you can just use `std::enable_if_t` directly, right?

Reply:
Good point; I've revised to do exactly that.  Thanks!